metricbeat/metricbeat.full.yml

```diff
@@ -64,8 +64,8 @@ metricbeat.modules:
   # if true, exports the CPU usage in ticks, together with the percentage values
   #cpu_ticks: false
 
-  # EXPERIMENTAL: cgroups can be enabled for the process metricset.
-  #cgroups: false
+  # Enable collection of cgroup metrics from processes on Linux.
+  #process.cgroups.enabled: true
```

Comment:
@andrewkroh Sounds like something we should discuss in general as it would also apply to all other beats. I kind of like the docs as it helps new users (I hope) but it is much more noisy, agree.